docs/whats_new.rst

```diff
@@ -17,6 +17,77 @@ in specific versions.
 
 .. towncrier release notes start
 
+.. _vp2p7p0:
+
+v2.7.0
+------
+
+This release comes with support for `python 3.11`_ and :ddocs:`new selects <change-log#new-select-menu-components>`.
+
+.. _python 3.11: https://docs.python.org/3/whatsnew/3.11.html
```

Comment:
This should be linked through intersphinx

Reply:
https://github.com/DisnakeDev/disnake/pull/841/commits/b6d711d6777ba0e42aa6a3e9e930cdd75042bcfa